tests/zkevm/test_worst_compute.py

```diff
@@ -515,6 +515,44 @@ def jump_seq():
     )
 
 
+@pytest.mark.zkevm
+@pytest.mark.valid_from("Cancun")
+@pytest.mark.slow
+def test_worst_jumpis(
+    state_test: StateTestFiller,
+    pre: Alloc,
+):
+    """Test running a JUMPI-intensive contract."""
+    env = Environment()
+
+    def jumpi_seq():
+        return Op.JUMPI(Op.ADD(Op.PC, 1), Op.PUSH0)
+
+    bytes_per_seq = len(jumpi_seq())
+    seqs_per_call = MAX_CODE_SIZE // bytes_per_seq
+
+    # Create and deploy the jump-intensive contract
+    jumpis_code = sum([jumpi_seq() for _ in range(seqs_per_call)])
```

Comment:
```suggestion
    jumpis_code = jumpi_seq() * seqs_per_call
```

I believe this also works :smile: :+1: